### ISSUE TITLE
Color contrast ratio of tree view description text is 4.381:1 which is less than 4.5:1: A11y_Visual Studio Code Client_Default screen_Color Contrast

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode#246455]
### GIT Hub Tags:
#A11yTCS; #A11ySev3; #Visual Studio Code Client; #BM-VisualStudioCodeClient-Win32-Jan2024; #DesktopApp; #A11yMAS; #WCAG1.4.3; #ColorContrast; #Win11; #FTP;

### Environment and OS details:
Application name: Visual studio code

### Reproduction Steps:

1. Open Visual studio code
2. Tab till Azure tab and press enter.
3. "Remote" description in the Resources tree view
4. Open accessibility insights for windows and check color contrast ratio of "Remote" text.
5. Observe that "Remote" description in the Resources tree view the Color contrast ratio of "Remote" text is 4.381:1 which is less than 4.5:1 or not.

### Actual behavior:
Color contrast ratio of "Remote" text is 4.381:1 which is less than 4.5:1

### Expected behavior:
Color contrast ratio of "Remote" text should be greater than or equals to 4.5:1

### User Impact:
Low vision users will not be able to see "Remote" text if color contrast ratio will be less than 4.5:1.

### Attachment:

![Image](https://github.com/user-attachments/assets/caf0ecd4-9bdc-4f52-8db2-f8698f4d4323)

**Comment by kupatkar99** [comment]
https://github.com/microsoft/vscode-azurefunctions/issues/4481rimary Bug ID: https://github.com/microsoft/vscode-azurefunctions/issues/4481
